AggregateMetrics/AggregateMetrics/AzureWebApp/RateCounter.cs

```diff
@@ -45,9 +45,9 @@ public MetricTelemetry GetValueAndReset()
                 return metric;
             }
 
-            metric.Value = ((double)this.lastValue - CacheHelper.Instance.GetCounterValue(this.name)) / (System.DateTime.Now.Second - this.dateTime.Second);
+            metric.Value = ((double)this.lastValue - CacheHelper.Instance.GetCounterValue(this.name)) / (System.DateTimeOffset.Now.Second - this.dateTime.Second);
             this.lastValue = CacheHelper.Instance.GetCounterValue(this.name);
-            this.dateTime = System.DateTime.Now;
+            this.dateTime = System.DateTimeOffset.Now;
```

Comment:
it would be good to take "Now" once in the beginning of the method and use it in formulae above as well as a new value